simphony/io/cuds_file.py

```diff
@@ -51,7 +53,6 @@ def open(cls, filename, mode="a", title=''):
         filename : str
             Name of file to be opened.
 
-
```

Comment:
styling nit: Please revert this change, I do not see a reason for having two spaces